src/index.html

```diff
@@ -6,9 +6,43 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="Moyo"
+        class="logo__img">
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo"
          class="logo__img"
        >
```
Check please in all places